lib/LedgerSMB/Template/HTML.pm

```diff
@@ -34,6 +34,8 @@ sub escape {
     return undef unless defined $vars;
     #$vars = encode_entities($vars);
     $vars = escape_html($vars);
+    #Allow back only a few decorations.
+    $vars =~ s~&lt;(/?[biu]|br)&gt;~<$1>~g;
```

Comment:
And in both options, when will the unescaped string will be processed? The only processing happens before it reaches the Template, so we will only get the format string back, not the processed one.